style.css

```diff
@@ -0,0 +1,628 @@
+    font-family: "Lato", Arial, sans-serif;
+    font-weight: bold;
+    width: 100%;
+    font-size: 10px;
+}
+
+.header__navbar{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 1020px;
+    margin: 0 auto;
+    position: relative;
+}
+
+.header__navbar::before,
+.header__navbar::after{
+    content:'';
+    width: 40%;
+    height: 1px;
+    background-color: #e4e9eb;
+    position: absolute;
+    bottom: -9px;
+}
+
+.header__navbar::after{
+    left: 31px;
+}
+
+.header__navbar::before{
+    right: 31px;
+}
+
+.navbar__logo{
+    display: inline-block;
+    padding: 0 12px;
+    position: relative;
+    margin: 10px 12px 0;
+    height: 56px;
+    width: 64px;
+}
+
+.navbar__logo img{
+    z-index: 3;
+}
+
+.navbar__logo::before{
+    content: "";
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 27px 46px 0 46px;
+    border-color: #323232 transparent transparent transparent;
+    position: absolute;
+    left: 0;
+    bottom: -23px;
+    z-index: -2;
+}
+
+.navbar__list{
+    display: flex;
+    align-items: center;
+    list-style: none;
+}
+
+.navbar__item{
+    display: inline-block;
+    text-transform: uppercase;
+    margin: 6px 12px 0;
+    position: relative;
+}
+
+.navbar__item:nth-of-type(2):before,
+.navbar__item:nth-of-type(5):before{
+    content: "";
+    width: 0px;
+    height: 0px;
+    color: white;
+    position: absolute;
+    top: 6px;
+    left: -13px;
+    border: 1px solid #474c5f;
+    border-radius: 50%;
+}
+
+.navbar__link{
+    color: #fff;
+    text-decoration: none;
+}
+
+.navbar__item .navbar__link{
+    transition: color .2s ease-out;
+}
+
+.navbar__item_active .navbar__link, 
+.navbar__item:hover .navbar__link{
+    color: #2eb398;
+}
+
+/* slider */
+
+.slider{
+    width: 1020px;
+    margin: 91px auto 117px;
+}
+
+.slider__container::before{
+    content: '';
+    width: 94%;
+    height: 1px;
+    position: absolute;
+    background-color: #e4e9eb;
+    bottom: -106px;
+    left: 50%;
+    -moz-transform: translateX(-50%);
+    -webkit-transform: translateX(-50%); 
+    transform: translateX(-50%);
+}
+
+/* hexagon */
+
+.hexagon{ 
+    display: inline-block;
+}
+
+.hexagon__layer1{
+    width: 179px;
+    height: 206px;
+    -moz-transform: rotate(-120deg);
+    -webkit-transform: rotate(-120deg);
+    transform: rotate(-120deg);
+    overflow: hidden;
+}
+
+.hexagon__layer2,
+.hexagon__layer3{
+    width: 100%;
+    height: 100%;
+    position: relative;
+    -moz-transform: rotate(-60deg);
+    -webkit-transform: rotate(-60deg);
+    transform: rotate(-60deg);
+    overflow: hidden;
+}
+
+.hexagon__image{
+    width: 123%;
+    position: absolute;
+    top: -4px;
+    left: -21px;
+    -moz-transform: rotate(-120deg);
+    -webkit-transform: rotate(-120deg);
+    transform: rotate(-120deg);
+    transition: width 0.5s ease-in, 
+                top 0.5s ease-in, 
+                left 0.5s ease-in;
+    cursor: pointer;
```

Comment:
cursor: pointer; is true for image in slider, but you also set this class for picture in "about me" and don't have effect for this image

Reply:
plus another style... you don't have effect